### PR TITLE
Pinned yarn and node using Volta and updated package.json

### PR DESCRIPTION
**Pull request by Maheima**

### CODE REVIEW

File: src/screens/AuthScreen/AuthScreen.tsx [hunk 1]
```diff
@@ -65,23 +71,23 @@ const AuthScreen = () => {
                 } catch (err) {
                   setLoggedInUserData(null);
                 }
-              } else if(url.indexOf("?") > 0) {
-                let uri = url.substring(0, url.indexOf("?"))
-                setAdressbarURL(uri)
+              } else if (url.indexOf('?') > 0) {
```
Comment (shreya-mishra) [inline src/screens/AuthScreen/AuthScreen.tsx:74]: why these unnecessary changes are there?

**Review by shreya-mishra — COMMENTED** [review]
Please don't update the unnecessary changes 😅  like these double quotes are converted to a single quote

File: src/screens/AuthScreen/AuthScreen.tsx (hunk 1 above)
Comment (Maheima) [reply src/screens/AuthScreen/AuthScreen.tsx:74]: Oh, I didn't make them. Wondering why this happened

Comment (shreya-mishra) [reply src/screens/AuthScreen/AuthScreen.tsx:74]: check prettier settings .

**Review by shreya-mishra — DISMISSED** [review]
LGTM :)

**Review by shreya-mishra — APPROVED** [review]
LGTM
